librad/tests/propagation_basic.rs

```diff
@@ -41,13 +41,21 @@ async fn can_clone() {
     testnet::run_on_testnet(peers, async move |mut apis| {
         let peer1 = apis.pop().unwrap();
         let peer2 = apis.pop().unwrap();
-
-        let alice = Alice::new(peer1.public_key());
-        let mut radicle = Radicle::new(&alice);
-        radicle
-            .sign(peer1.key(), &Signatory::User(alice.urn()), &alice)
-            .await
+        let mut cache = EntityMemoryCache::default();
+
+        let mut alice = Alice::new(peer1.public_key());
+        alice.sign_owned(peer1.key()).unwrap();
+        let verified_alice = alice
+            .clone()
+            .check_signatures()
+            .unwrap()
+            .check_signatures_ownership(&cache)
+            .unwrap()
+            .check_update(&None)
```

Comment:
I think it'd be nice to have a top-level function that can do all this to get a verified entity. Remembering to do all the checks seems verbose.

Reply:
Done, even before reading the review.
The high level API was already in the works when we did the walkthrough but then I got ill...